src/views/templates/actions/components/EditBootSourceModal.tsx

```diff
@@ -62,29 +64,28 @@ const EditBootSourceModal: FC<EditBootSourceModalProps> = ({
         isOpen={isOpen}
         onClose={onClose}
       >
-        <Alert isInline className="margin-bottom-md" variant="warning" title={t('Warning')}>
-          <Trans ns="plugin__kubevirt-plugin">
+        <Trans ns="plugin__kubevirt-plugin">
```

Comment:
im not sure Trans is needed here

Reply:
You are right, this line of the code can be in its original place.